### ISSUE TITLE
Reordering tabs refreshes them and signs me out

### ISSUE THREAD

**Issue by Hadi-Osman** [mbnuqw/sidebery#1010]
Reordering tabs in sidebery beta v5.0.0b31 causes the tab to be refreshed and I get signed out of all accounts on this tab as if all session cookies are cleared (manually refreshing the page again doesn't auto sign in either) it's like an incognito/private tab was opened. 

## Steps to reproduce
- Open Firefox in [new profile](https://support.mozilla.org/en-US/kb/profile-manager-create-remove-switch-firefox-profiles)
- Install Sidebery
- Open multiple tab (for example youtube and reddit)
- Reordering youtube tab for example causes the tab to be refreshed and signed out of all accounts on this particular tab that has been reordered.

## Expected behavior
Reordering the tab shouldn't affect it

## Actual behavior
Tab reordering signs out of all accounts on the reordered tab.

## Debug info

<details>
<summary><b>Addon data</b></summary>
     
```
{
  "addonVersion": "5.0.0b31",
  "firefoxVersion": "111.0.1",
  "settings": {
    "nativeScrollbars": false,
    "nativeScrollbarsThin": true,
    "nativeScrollbarsLeft": false,
    "selWinScreenshots": false,
    "updateSidebarTitle": true,
    "markWindow": true,
    "markWindowPreface": "[S] ",
    "ctxMenuNative": true,
    "ctxMenuRenderInact": true,
    "ctxMenuRenderIcons": true,
    "ctxMenuIgnoreContainers": "",
    "navBarLayout": "horizontal",
    "navBarInline": false,
    "navBarSide": "left",
    "hideAddBtn": false,
    "hideSettingsBtn": false,
    "navBtnCount": true,
    "hideEmptyPanels": true,
    "navActTabsPanelLeftClickAction": "none",
    "navActBookmarksPanelLeftClickAction": "none",
    "navTabsPanelMidClickAction": "none",
    "navBookmarksPanelMidClickAction": "none",
    "navSwitchPanelsWheel": true,
    "groupLayout": "grid",
    "skipEmptyPanels": false,
    "dndTabAct": false,
    "dndTabActDelay": 750,
    "dndTabActMod": "none",
    "dndExp": "none",
    "dndExpDelay": 750,
    "dndExpMod": "none",
    "dndOutside": "win",
    "searchBarMode": "static",
    "warnOnMultiTabClose": "collapsed",
    "activateLastTabOnPanelSwitching": true,
    "activateLastTabOnPanelSwitchingLoadedOnly": true,
    "tabRmBtn": "always",
    "hideInact": false,
    "activateAfterClosing": "next",
    "activateAfterClosingStayInPanel": false,
    "activateAfterClosingGlobal": false,
    "activateAfterClosingNoFolded": true,
    "activateAfterClosingNoDiscarded": true,
    "askNewBookmarkPlace": true,
    "tabsRmUndoNote": true,
    "nativeHighlight": false,
    "tabsUnreadMark": false,
    "tabsUpdateMark": "all",
    "tabsUpdateMarkFirst": true,
    "tabsReloadLimit": 3,
    "tabsReloadLimitNotif": false,
    "showNewTabBtns": true,
    "newTabBarPosition": "bottom",
    "tabsPanelSwitchActMove": true,
    "tabsUrlInTooltip": "full",
    "openSubPanelOnMouseHover": false,
    "selectActiveTabFirst": true,
    "moveNewTabPin": "end",
    "moveNewTabParent": "last_child",
    "moveNewTabParentActPanel": false,
    "moveNewTab": "end",
    "moveNewTabActivePin": "start",
    "pinnedTabsPosition": "panel",
    "pinnedTabsList": true,
    "pinnedAutoGroup": false,
    "tabsTree": true,
    "groupOnOpen": true,
    "tabsTreeLimit": 5,
    "hideFoldedTabs": false,
    "hideFoldedParent": "none",
    "autoFoldTabs": false,
    "autoFoldTabsExcept": "none",
    "autoExpandTabs": false,
    "rmChildTabs": "none",
    "tabsChildCount": true,
    "tabsLvlDots": false,
    "discardFolded": false,
    "discardFoldedDelay": 0,
    "discardFoldedDelayUnit": "sec",
    "tabsTreeBookmarks": true,
    "treeRmOutdent": "first_child",
    "colorizeTabs": false,
    "colorizeTabsSrc": "domain",
    "colorizeTabsBranches": true,
    "colorizeTabsBranchesSrc": "domain",
    "inheritCustomColor": true,
    "warnOnMultiBookmarkDelete": "collapsed",
    "autoCloseBookmarks": false,
    "autoRemoveOther": false,
    "highlightOpenBookmarks": false,
    "activateOpenBookmarkTab": false,
    "showBookmarkLen": false,
    "bookmarksRmUndoNote": true,
    "loadBookmarksOnDemand": true,
    "pinOpenedBookmarksFolder": true,
    "loadHistoryOnDemand": true,
    "fontSize": "xxs",
    "animations": true,
    "animationSpeed": "fast",
    "theme": "plain",
    "density": "default",
    "colorScheme": "ff",
    "sidebarCSS": true,
    "groupCSS": false,
    "snapNotify": false,
    "snapExcludePrivate": false,
    "snapInterval": 30,
    "snapIntervalUnit": "min",
    "snapLimit": 50,
    "snapLimitUnit": "snap",
    "hScrollAction": "switch_panels",
    "navSwitchPanelsDelay": 128,
    "scrollThroughTabs": "panel",
    "scrollThroughVisibleTabs": false,
    "scrollThroughTabsSkipDiscarded": false,
    "scrollThroughTabsExceptOverflow": true,
    "scrollThroughTabsCyclic": false,
    "scrollThroughTabsScrollArea": 0,
    "autoMenuMultiSel": true,
    "multipleMiddleClose": false,
    "longClickDelay": 500,
    "wheelThreshold": false,
    "wheelThresholdX": 10,
    "wheelThresholdY": 60,
    "tabDoubleClick": "exp",
    "tabsSecondClickActPrev": false,
    "tabsSecondClickActPrevPanelOnly": false,
    "shiftSelAct": true,
    "activateOnMouseUp": false,
    "tabLongLeftClick": "none",
    "tabLongRightClick": "pin",
    "tabCloseMiddleClick": "discard",
    "tabsPanelLeftClickAction": "none",
    "tabsPanelDoubleClickAction": "tab",
    "tabsPanelRightClickAction": "menu",
    "tabsPanelMiddleClickAction": "tab",
    "newTabMiddleClickAction": "new_child",
    "bookmarksLeftClickAction": "open_in_new",
    "bookmarksLeftClickActivate": false,
    "bookmarksLeftClickPos": "default",
    "bookmarksMidClickAction": "open_in_new",
    "bookmarksMidClickActivate": false,
    "bookmarksMidClickPos": "default",
    "syncName": "Firefox",
    "syncSaveSettings": true,
    "syncSaveCtxMenu": true,
    "syncSaveStyles": true,
    "syncSaveKeybindings": true
  },
  "permissions": {
    "allUrls": true,
    "tabHide": true,
    "clipboardWrite": true,
    "webRequest": true,
    "webRequestBlocking": true
  },
  "storage": {
    "size": "196 kb",
    "props": {
      "containers": "1.58 kb",
      "contextMenu": "898 b",
      "favDomains": "6.93 kb",
      "favHashes": "819 b",
      "favicons": "60.2 kb",
      "lastSnapTime": "13 b",
      "profileID": "14 b",
      "settings": "3.86 kb",
      "sidebar": "1.35 kb",
      "sidebarCSS": "13.5 kb",
      "snapshots": "92.5 kb",
      "tabsDataCache": "14.5 kb",
      "ver": "10 b"
    }
  },
  "sidebar": {
    "panels": {
      "c42j05-ePpdh": {
        "type": 1,
        "id": "c42j05-ePpdh",
        "name": "len: 9",
        "iconSVG": "icon_bookmarks",
        "iconIMGSrc": "",
        "iconIMG": "",
        "color": "toolbar",
        "lockedPanel": false,
        "tempMode": false,
        "skipOnSwitching": false,
        "rootId": "root________",
        "viewMode": "tree",
        "autoConvert": false
      },
      "QixI5PDp6z0_": {
        "type": 2,
        "id": "QixI5PDp6z0_",
        "name": "len: 4",
        "color": "toolbar",
        "iconSVG": "icon_code",
        "iconIMGSrc": "",
        "iconIMG": "",
        "lockedPanel": true,
        "skipOnSwitching": false,
        "noEmpty": false,
        "newTabCtx": "none",
        "dropTabCtx": "firefox-container-1",
        "moveTabCtx": "firefox-container-1",
        "moveTabCtxNoChild": true,
        "bookmarksFolderId": -1,
        "newTabBtns": []
      },
      "eRIWuhaghhdv": {
        "type": 2,
        "id": "eRIWuhaghhdv",
        "name": "len: 4",
        "color": "orange",
        "iconSVG": "briefcase",
        "iconIMGSrc": "",
        "iconIMG": "",
        "lockedPanel": false,
        "skipOnSwitching": false,
        "noEmpty": false,
        "newTabCtx": "firefox-container-2",
        "dropTabCtx": "firefox-container-2",
        "moveTabCtx": "firefox-container-2",
        "moveTabCtxNoChild": true,
        "bookmarksFolderId": -1,
        "newTabBtns": []
      },
      "ypDCyg9TRYUO": {
        "type": 2,
        "id": "ypDCyg9TRYUO",
        "name": "len: 5",
        "color": "yellow",
        "iconSVG": "icon_archive",
        "iconIMGSrc": "",
        "iconIMG": "",
        "lockedPanel": false,
        "skipOnSwitching": false,
        "noEmpty": false,
        "newTabCtx": "firefox-container-3",
        "dropTabCtx": "firefox-container-3",
        "moveTabCtx": "none",
        "moveTabCtxNoChild": true,
        "bookmarksFolderId": -1,
        "newTabBtns": []
      }
    },
    "nav": [
      "c42j05-ePpdh",
      "QixI5PDp6z0_",
      "eRIWuhaghhdv",
      "ypDCyg9TRYUO"
    ]
  },
  "containers": [
    {
      "id": "firefox-container-1",
      "cookieStoreId": "firefox-container-1",
      "name": "8",
      "icon": "...",
      "color": "blue",
      "colorCode": "#37adff",
      "proxified": false,
      "proxy": null,
      "includeHostsActive": false,
      "includeHosts": "",
      "excludeHostsActive": false,
      "excludeHosts": "",
      "userAgentActive": false,
      "userAgent": ""
    },
    {
      "id": "firefox-container-2",
      "cookieStoreId": "firefox-container-2",
      "name": "4",
      "icon": "...",
      "color": "orange",
      "colorCode": "#37adff",
      "proxified": false,
      "proxy": null,
      "includeHostsActive": false,
      "includeHosts": "",
      "excludeHostsActive": false,
      "excludeHosts": "",
      "userAgentActive": false,
      "userAgent": ""
    },
    {
      "id": "firefox-container-3",
      "cookieStoreId": "firefox-container-3",
      "name": "7",
      "icon": "...",
      "color": "green",
      "colorCode": "#37adff",
      "proxified": false,
      "proxy": null,
      "includeHostsActive": false,
      "includeHosts": "",
      "excludeHostsActive": false,
      "excludeHosts": "",
      "userAgentActive": false,
      "userAgent": ""
    },
    {
      "id": "firefox-container-4",
      "cookieStoreId": "firefox-container-4",
      "name": "8",
      "icon": "...",
      "color": "pink",
      "colorCode": "#37adff",
      "proxified": false,
      "proxy": null,
      "includeHostsActive": false,
      "includeHosts": "",
      "excludeHostsActive": false,
      "excludeHosts": "",
      "userAgentActive": false,
      "userAgent": ""
    },
    {
      "id": "firefox-container-6",
      "cookieStoreId": "firefox-container-6",
      "name": "15",
      "icon": "...",
      "color": "blue",
      "colorCode": "#37adff",
      "proxified": false,
      "proxy": null,
      "includeHostsActive": false,
      "includeHosts": "",
      "excludeHostsActive": false,
      "excludeHosts": "",
      "userAgentActive": false,
      "userAgent": ""
    }
  ],
  "sidebarCSSLen": "13189",
  "windows": [
    {
      "state": "maximized",
      "incognito": false,
      "tabsCount": 114
    }
  ],
  "bookmarks": {
    "bookmarksCount": 843,
    "foldersCount": 118,
    "separatorsCount": 0,
    "maxDepth": 7
  }
}
```
</details>

<details>
<summary><b>Logs</b></summary>

```
[bg] Tabs.injectUrlPageScript: Cannot inject script, tabId: 132 Error: An unexpected error occurred chunk-M3N7CHPP.js:2:7531
```
</details>

**Comment by mbnuqw** [comment]
This happens because you set the "Reopen tab that was dropped to this panel in container..." option in panel config.

For example, if you open a new tab with `ctrl+T`, Firefox will create this tab in the default container. Then, when you drag and drop that tab, Sidebery will reopen it in the target container, hence cookies in that tab will be different. This issue will be fixed in the next beta release (#305).

You can also open a new tab with the "Open new tab in active panel" Sidebery keyboard shortcut, which will use a container configured for new tabs in the panel config.

**Comment by drkhn1234** [comment]
@Hadi-Osman , you can use shortcuts from Sidebery Settings > Keybindings > "Moving tabs": "Move tabs up" and "Move tabs down". Using them will allow you to move tabs just like with drag-n-dropping with mouse, but won't change the container.
